packages/reactive-element/package.json

```diff
@@ -79,23 +79,135 @@
     }
   },
   "scripts": {
-    "build": "npm run clean && npm run build:ts && rollup -c && npm run build:babel",
-    "build:watch": "rollup -c --watch",
-    "build:ts": "tsc --build && treemirror development . '**/*.d.ts{,.map}'",
-    "build:ts:watch": "tsc --build --watch",
-    "build:babel": "babel --extensions '.ts' src/test/decorators  --out-dir development/test/decorators-babel",
-    "check-version": "node scripts/check-version-tracker.js",
-    "checksize": "rollup -c --environment=CHECKSIZE",
-    "clean": "rm -rf {css-tag,decorators,polyfill-support,reactive-controller,reactive-element}.{js,js.map,d.ts} test/ decorators/ development/ *.tsbuildinfo",
-    "dev": "scripts/dev.sh",
+    "build": "wireit",
+    "build:ts": "wireit",
+    "build:ts:types": "wireit",
+    "build:rollup": "wireit",
+    "build:babel": "wireit",
+    "check-version": "wireit",
+    "checksize": "wireit",
     "prepublishOnly": "npm run check-version",
-    "publish-dev": "npm test && VERSION=${npm_package_version%-*}-dev.`git rev-parse --short HEAD` && npm version --no-git-tag-version $VERSION && npm publish --tag dev",
-    "regen-package-lock": "rm -rf node_modules package-lock.json; npm install",
-    "test": "npm run test:dev && npm run test:prod",
-    "test:dev": "cd ../tests && npx wtr '../reactive-element/development/**/*_test.(js|html)'",
-    "test:prod": "MODE=prod npm run test:dev",
-    "test:prod:watch": "MODE=prod npm run test:dev -- --watch",
-    "test:watch": "npm run test:dev -- --watch"
+    "test": "wireit",
+    "test:dev": "wireit",
+    "test:prod": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:rollup",
+        "build:babel",
```

Comment:
`build:babel` is only needed for testing, right? So we could remove it from here.